spec_mode/nirspec_fs.ipynb

```diff
@@ -122,7 +122,9 @@
    "cell_type": "code",
```

Comment:
<p>I un-commented this cell since any changes would probably be merged after the JWebbinars have concluded.</p><p> </p><p>Another way to keep a cell visible while preventing it from executing is to add <code>%%script false --no-raise-error</code> at the top. This is "Jupyter magic" that could help avoid the need for large block comments and make the code more readable. If you go this route, it would be useful to add a note above the cell in Markdown to ensure the reader understands what's going on and how to change it.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/jwebbinar_prep/pull/44/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>